dashboard/test/unit/test_cbroker.js

```diff
@@ -253,5 +254,25 @@ suite('cbroker', function () {
 
     });
 
+    test('should_filter_region_list_and_remove_disabled_region', function () {
```

Comment:
this data only is a sample data. The input data not is deterministic for the new functionality. You could see the same in other test: (e.g. should_return_a_json_with_all_regions_and_status).